teamA/lib/Views/send_essay_to_moodle.dart

```diff
@@ -74,11 +60,69 @@ class _EssayAssignmentSettingsState extends State<EssayAssignmentSettings> {
   List<String> minutes =
       List.generate(60, (index) => index.toString().padLeft(2, '0'));
 
-  TextEditingController _courseNameController = TextEditingController();
-  TextEditingController _assignmentNameController = TextEditingController();
+  final TextEditingController _assignmentNameController =
+      TextEditingController();
 
   // Quill Editor controller
-  quill.QuillController _quillController = quill.QuillController.basic();
+  final quill.QuillController _quillController = quill.QuillController.basic();
+
+  // List of courses with their corresponding course IDs
+  Map<String, String> courses = {
```

Comment:
Should we not be getting this programmatically?

Reply:
I need to get courses in the dropdown list for my part. If the courses section is ready, I can incorporate it into my code.